IBX-7547: Labels & screenshot corrected (#289)
ibexa/documentation-user: docs/getting_started/get_started.md

![User preferences menu](img/user_preferences.png)

Here you can change your user password and define your user preferences,

> such as preferred timezone, short of full date and time format, or Back Office language.

Reviewer: ```suggestion
such as preferred timezone, short and full date and time format, or Back Office language.
```
Both options are there to set

> such as preferred timezone, short of full date and time format, or Back Office language.

Reviewer 2: Still trying to remove an "and"
Inspired by https://learn.microsoft.com/en-us/system-center/orchestrator/standard-activities/format-date-time https://www.ibm.com/docs/fr/i/7.5?topic=items-working-date-time-data-types

```suggestion
such as preferred timezone, short and full date time format, or Back Office language.
```
or
```suggestion
such as preferred timezone, short and full date/time format, or Back Office language.
```

> such as preferred timezone, short of full date and time format, or Back Office language.

Reviewer: The last one!

> such as preferred timezone, short of full date and time format, or Back Office language.

Reviewer 2: Do we really need to know that there will be short and full in this early list of examples?

```suggestion
such as preferred timezone, date and time formats, or Back Office language.
```
